Compatibility with openssl-devel from RedHat systems ?

Hi everyone,

I'm trying to build servo on a Fedora 26 with openssl-devel 1.1.0e.

It appears that a function is missing, "./mach build --release" returns :
```
error[E0425]: cannot find function `EC_GROUP_get_curve_GF2m` in module `ffi`
  --> .../servo/.cargo/registry/src/github.com-1ecc6299db9ec823/openssl-0.9.10/src/ec.rs:76:22
   |
76 |             cvt(ffi::EC_GROUP_get_curve_GF2m(self.as_ptr(),
   |                      ^^^^^^^^^^^^^^^^^^^^^^^ did you mean `EC_GROUP_get_curve_GFp`?

error: aborting due to previous error
error: Could not compile `openssl`.
```

Is there any way to build it on my Fedora ?
Thank's.

--- a/servo-tidy.toml
+++ b/servo-tidy.toml
@@ -31,7 +31,7 @@ num = []
 
 [ignore]
 # Ignored packages with duplicated versions
-packages = []
+packages = ["bitflags"]
 # Files that are ignored for all tidy and lint checks.
 files = [
   # Helper macro where actually a pseudo-element per line makes sense.

